types/convict/index.d.ts

```diff
@@ -20,7 +20,7 @@ declare namespace convict {
     interface ValidateOptions {
         /**
          * If set to warn, any properties specified in config files that are not declared in
-         * the schema will print a warning. This is the default behavior. If set to strict,
+         * the schema will print a warning. This is the dƒefault behavior. If set to strict,
```

Comment:
typo?

Reply:
My shift key isn't working 😢. Thanks for catching it.